src/pages/workspaces/workspace/analysis/Analyses.test.ts

```diff
@@ -0,0 +1,375 @@
+type AjaxContract = ReturnType<typeof Ajax>;
+type AjaxMetricsContract = AjaxContract['Metrics']
+type OuterWorkspacesContract = AjaxContract['Workspaces']
+type InnerWorkspacesContract = AjaxContract['Workspaces']['workspace']
+const mockInnerWorkspaces = jest.fn().mockReturnValue({
+  listActiveFileTransfers: jest.fn()
+}) as InnerWorkspacesContract
+const mockOuterWorkspaces: Partial<OuterWorkspacesContract> = {
+  workspace: mockInnerWorkspaces
+}
+
+const mockMetrics: Partial<AjaxMetricsContract> = {
+  captureEvent: (event, details) => watchCaptureEvent(event, details)
+}
+
+const defaultAjaxImpl: Partial<AjaxContract> = {
+  Workspaces: mockOuterWorkspaces as OuterWorkspacesContract,
+  Metrics: mockMetrics as AjaxMetricsContract
+}
+
+describe('Analyses', () => {
+  beforeEach(() => {
+    // Arrange
+    asMockedFn(useAnalysisFiles).mockImplementation(() => defaultUseAnalysisStore)
+    asMockedFn(notebookLockHash).mockImplementation(() => Promise.resolve('testhash'))
+    asMockedFn(findPotentialNotebookLockers).mockImplementation(() => Promise.resolve({}))
+    asMockedFn(Ajax).mockImplementation(() => defaultAjaxImpl as AjaxContract)
+    asMockedFn(getLocalPref).mockImplementation(() => undefined)
+  })
+
+  it('loads properly with no files', async () => {
+    // Act
+    await act(async () => {
+      await render(h(BaseAnalyses, defaultAnalysesProps))
+    })
+
+    // Assert
+    screen.getByText('Start')
+    // Banner text and images
+    screen.getByText('A place for all your analyses')
+    screen.getByAltText('Jupyter')
+    screen.getByAltText('RStudio Bioconductor')
+    screen.getByAltText('Galaxy')
+  })
+
+  it('loads properly with files for a google workspace', async () => {
+    // Arrange
+    const fileName1 = 'file1.ipynb'
+    const fileName2 = 'file2.ipynb'
+    const files = [getFileFromPath(`test/${fileName1}` as AbsolutePath), getFileFromPath(`test/${fileName2}` as AbsolutePath)]
+
+    asMockedFn(useAnalysisFiles).mockImplementation(() => ({
+      ...defaultUseAnalysisStore,
+      loadedState: { state: files, status: 'Ready' }
+    }))
+
+    // Act
+    await act(async () => {
+      await render(h(BaseAnalyses, defaultAnalysesProps))
+    })
+
+    // Assert
+    screen.getByText('Start')
+    // Table headings
+    screen.getByText('Application')
+    screen.getByText('Name')
+    screen.getByText('Last Modified')
+    screen.getByPlaceholderText('Search analyses')
+    screen.getByText(fileName1)
+    screen.getByText(fileName2)
+  })
+
+  it('loads properly with files for an azure workspace', async () => {
+    // Arrange
+    const fileName1 = 'file1.ipynb'
+    const fileName2 = 'file2.ipynb'
+    const files = [getFileFromPath(`test/${fileName1}` as AbsolutePath), getFileFromPath(`test/${fileName2}` as AbsolutePath)]
+
+    asMockedFn(useAnalysisFiles).mockImplementation(() => ({
+      ...defaultUseAnalysisStore,
+      loadedState: { state: files, status: 'Ready' }
```

Comment:
nit: for discriminating-union types like LoadedState, it is best practice to list the discriminating prop/s first, status in this case.

Reply:
same nit for a few tests after this